### PR TITLE
Fix asking for permissions

### PR DESCRIPTION
**Pull request by Rados13**

### CODE REVIEW

File: assets/js/utils/rtcUtils.ts [hunk 1]
```diff
@@ -266,3 +267,19 @@ const removeMergedStream = () => {
 const removeStream = (name: string) => {
   delete presenterStreams[name];
 };
+
+const askForPermissions = async (): Promise<void> => {
+  const hasVideoInput: boolean = (await navigator.mediaDevices.enumerateDevices()).some(
+    (device) => device.kind === "videoinput"
+  );
+
+  let tmpVideoStream = await navigator.mediaDevices.getUserMedia({
```
Comment (Karolk99) [inline assets/js/utils/rtcUtils.ts:276]: Can it be const ?

File: config/runtime.exs [hunk 2]
```diff
@@ -80,10 +80,10 @@ host = System.get_env("VIRTUAL_HOST", "localhost")
 port = 4000
 
 config :membrane_live, MembraneLive.Repo,
-  username: System.get_env("PGUSER", "swm"),
-  password: System.get_env("PGPASSWORD", "swm123"),
+  username: System.get_env("POSTGRES_USER", "swm"),
+  password: System.get_env("POSTGRES_PASSWORD", "swm123"),
   hostname: System.get_env("PGHOST", "localhost"),
```
Comment (Karolk99) [inline config/runtime.exs:85]: in `.env.sample` we have `VIRTUAL_HOST=localhost`

File: docker-compose.yml [hunk 3]
```diff
@@ -28,9 +37,9 @@ services:
     depends_on:
       - membrane-live-db
     environment:
-      PGUSER: "${POSTGRES_USER:-membrane}"
-      PGPASSWORD: "${POSTGRES_PASSWORD}"
-      PGDATABASE: "${POSTGRES_DB:-live_db}"
+      POSTGRES_USER: "${POSTGRES_USER:-membrane}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      POSTGRES_DB: "${POSTGRES_DB:-live_db}"
       PGHOST: localhost
```
Comment (Karolk99) [inline docker-compose.yml:43]: same as above

**Review by Karolk99 — APPROVED** [review]

File: config/runtime.exs (hunk 2 above)
Comment (piotrekzyla) [reply config/runtime.exs:85]: `VIRTUAL_HOST` is something else but I agree PGHOST should become `POSTGRES_HOST` for consistency

File: config/runtime.exs [hunk 4]
```diff
@@ -80,10 +80,10 @@ host = System.get_env("VIRTUAL_HOST", "localhost")
...
-  database: System.get_env("PGDATABASE", "membrane_live_db"),
+  database: System.get_env("POSTGRES_DB", "membrane_live_db"),
   port: System.get_env("PGPORT", "5432"),
```
Comment (piotrekzyla) [inline config/runtime.exs:87]: `PGPORT` -> `POSTGRES_PORT`

**Review by piotrekzyla — COMMENTED** [review]
Changes to `.env.sample` should go along this changes

**Review by piotrekzyla — APPROVED** [review]

**Pull request merged**